jenkins/Jenkinsfile

```
@@ -503,28 +501,26 @@ timestamps {
                             if (bundleZip.length > 0)
                                 bundleZipFilename = bundleZip [0].name
 
-                            withCredentials ([string (credentialsId: 'codesign_keychain_pw', variable: 'PRODUCTSIGN_KEYCHAIN_PASSWORD')]) {
-                                    sh ("${workspace}/xamarin-macios/jenkins/productsign.sh")
-                            }
-
-                            if (notarize_mac || notarize_ios) {
+                            if (isPr) {
+                              withCredentials ([string (credentialsId: 'codesign_keychain_pw', variable: 'PRODUCTSIGN_KEYCHAIN_PASSWORD')]) {
+                                sh ("${workspace}/xamarin-macios/jenkins/productsign.sh")
+                              }
+                            } else {
                                 try {
-                                    pkgs = []
+                                    pkgs = xiPackages + xmPackages
                                     if (fileExists('release-scripts')) {
                                         dir('release-scripts') {
                                             sh ('git checkout sign-and-notarized && git pull')
                                         }
                                     } else {
                                         sh ('git clone git@github.com:xamarin/release-scripts -b sign-and-notarized')
                                     }
-                                    if (notarize_mac)
-                                        pkgs = pkgs + xmPackages
-                                    if (notarize_ios)
-                                        pkgs = pkgs + xiPackages
                                     withCredentials([string(credentialsId: 'codesign_keychain_pw', variable: 'KEYCHAIN_PASS'), string(credentialsId: 'team_id', variable: 'TEAM_ID'), string(credentialsId: 'application_id', variable: 'APP_ID'), string(credentialsId: 'installer_id', variable: 'INSTALL_ID'), usernamePassword(credentialsId: 'apple_account', passwordVariable: 'APPLE_PASS', usernameVariable: 'APPLE_ACCOUNT')]) {
                                         sh (returnStatus: true, script: "security create-keychain -p ${env.KEYCHAIN_PASS} login.keychain") // needed to repopulate the keychain
                                         sh ("security unlock-keychain -p ${env.KEYCHAIN_PASS} login.keychain")
-                                        sh ("python release-scripts/sign_and_notarize.py -a ${env.APP_ID} -i ${env.INSTALL_ID} -u ${env.APPLE_ACCOUNT} -p ${env.APPLE_PASS} -t ${env.TEAM_ID} -d package/notarized -e ${entitlements} -k login.keychain " + pkgs.flatten().join(" "))
+                                        timeout(time: 90, unit: 'MINUTES') {
```

As with the other PR, lets me consistent.
```suggestion
                                        timeout (time: 90, unit: 'MINUTES') {
```